project/src/test/java/com/google/sps/data/PlacesScorerImplTest.java

```diff
@@ -37,34 +47,93 @@
         .setPriceLevel(4)
         .setLocation(new LatLng(32.08, 34.78))
         .setGoogleUrl("google.com")
-        .setPlaceId("placeid1")
+        .setPlaceId("placeid")
+        .setName("name")
         .setBusinessStatus(BusinessStatus.OPERATIONAL);
 
     private static final LatLng USER_LOCATION = new LatLng(33.12, 34.56);
-    private static final Place PLACE_1 = PLACE_BUILDER.setName("name1").setRating(3).build();
-    private static final Place PLACE_2 = PLACE_BUILDER.setName("name2").setRating(5).build();
-    private static final ImmutableList<Place> PLACES_TO_SCORE = ImmutableList.of(PLACE_1, PLACE_2);
+    private static final Place PLACE_RATING_3 = PLACE_BUILDER.setRating(3).build();
+    private static final Place PLACE_RATING_5 = PLACE_BUILDER.setRating(5).build();
+    private static final String[] PLACES_ADDRESSES = {"Place1 Address", "Place2 Address"};
```

Comment:
Why array and not ImmutableList?

Reply:
Because it is sent to DistanceMatrix constructor, which gets arrays and not lists.

Reply:
Ack, thanks.